src/components/Statistics.tsx

```diff
@@ -33,16 +34,14 @@ interface Props {
 const Statistics = ({ refresh }: Props) => {
   const { client } = useGlobalContext()
   const [hashrate, setHashrate] = useState('')
-  const [totalSupply, setTotalSupply] = useState('')
-  const [circulating, setCirculating] = useState('')
-  const [transactions, setTransactions] = useState('')
-  const [blocks, setBlocks] = useState('')
-  const [avgBlockTime, setAvgBlockTime] = useState('')
+  const [totalSupply, setTotalSupply] = useState<number>()
+  const [circulating, setCirculating] = useState<number>()
```

Comment:
Nit: I had kept all numbers as `string` to avoid precision loss as these numbers get higher.

Reply:
Since the numbers we receive from the API are integers and since these numbers won't ever reach the `MAX_SAFE_INTEGER` (9,007,199,254,740,991) (or will they?), then we can safely use `parseInt` to set the values in the state. Nothing actually changes, since you were also calling `parseInt` on the value stored in the state, before displaying them.